setup.py

```diff
@@ -19,6 +19,7 @@
     install_requires=[
         'six',
         'thriftpy',
+        'yelp-clog>=0.1.7',
```

Comment:
Strictly speaking, this PR requires staticconf as well, though I'd like it not to.